code/modules/mob/hear_say.dm

```diff
@@ -9,6 +9,8 @@
 			//Or someone snoring.  So we make it where they won't hear it.
 		return
 
+	message = capitalize(message)
```

Comment:
nononononononononononononononononononononononononononononononononononono 
jesus christ no

if you absolutely fucking have to, put this at https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/say.dm#L121